doc/examples.rst

```diff
@@ -0,0 +1,109 @@
+.. _examples:
```

Comment:
Copyright notice please.

Reply:
Note, when I say copyright, I mean copyright and licence.

Reply:
It's OK. I'm not taking you quite as literally today :-)